### PR TITLE
829272 - [others] Implicit Styling For Summaries.

### PR DESCRIPTION
**Pull request by sudarsanMS**
**Description**
We added the Group, Caption and Table Summary implicit styling in styling.

### CODE REVIEW

**Review by ashok-kuvaraja — APPROVED** [review]
The changes are fine.

File: MAUI/DataGrid/scrolling.md [hunk 1]
```diff
@@ -48,4 +48,41 @@ You can scroll programmatically to a particular column using the `SfDataGrid.Scr
...
 this.dataGrid.ScrollToColumnIndex(4, true, ScrollToPosition.MakeVisible);
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+## Scrollbar Visibility
+
+You can change the visibility of the horizontal and vertical scrollbar using the [SfDataGrid.HorizontalScrollBarVisibility]() and [SfDataGrid.VerticalScrollBarVisibility]() properties respectively. By default, the visibility of both the horizontal and vertical scrollbars is set to `Default`.
+
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/scrolling.md:56]: The visibility of the horizontal and vertical scrollbars can be customized using the [SfDataGrid.HorizontalScrollBarVisibility]() and [SfDataGrid.VerticalScrollBarVisibility]() properties. By default, the visibility of both the horizontal and vertical scrollbars is set to `ScrollBarVisibility.Default`.

The following code snippets demonstrate how to hide the vertical and horizontal scrollbars:

File: MAUI/DataGrid/scrolling.md [hunk 2]
```diff
@@ -48,4 +48,41 @@ You can scroll programmatically to a particular column using the `SfDataGrid.Scr
...
+{% highlight xaml %}
+    <sfgrid:SfDataGrid x:Name="dataGrid"                                       
+                       ItemsSource="{Binding OrdersInfo}"         
+                       HorizontalScrollBarVisibility="Never"
+                       VerticalScrollBarVisibility="Never">   
+    </sfgrid:SfDataGrid> 
+{% endhighlight %}
+
+{% highlight C# %}
+namespace GettingStarted
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/scrolling.md:67]: These codes will be enough.

dataGrid.HorizontalScrollBarVisibility = ScrollBarVisibility.Never;
dataGrid.VerticalScrollBarVisibility = ScrollBarVisibility.Never;

File: MAUI/DataGrid/scrolling.md [hunk 3]
```diff
@@ -48,4 +48,41 @@ You can scroll programmatically to a particular column using the `SfDataGrid.Scr
...
+            dataGrid.HorizontalScrollBarVisibility = ScrollBarVisibility.Never;
+            dataGrid.VerticalScrollBarVisibility = ScrollBarVisibility.Never;
+            this.Content = dataGrid;
+        }
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+N> These properties does not have any effect when the datagrid has no scrollable content in its respective direction. In such cases scroll bar will not be displayed
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/scrolling.md:88]: N> These properties do not have any effect when the datagrid has no scrollable content in its respective direction. In such cases, the scroll bar will not be displayed.

File: MAUI/DataGrid/scrolling.md (hunk 2 above)
Comment (sudarsanMS) [reply MAUI/DataGrid/scrolling.md:67]: updated

File: MAUI/DataGrid/scrolling.md (hunk 3 above)
Comment (sudarsanMS) [reply MAUI/DataGrid/scrolling.md:88]: updated

File: MAUI/DataGrid/scrolling.md (hunk 1 above)
Comment (sudarsanMS) [reply MAUI/DataGrid/scrolling.md:58]: updated

**Review by FarjanaParveen — APPROVED** [review]
This is fine

**Pull request merged**